### ISSUE TITLE
provider failed with terraform 1.5.3

### ISSUE THREAD

**Issue by samuev** [jfrog/terraform-provider-project#79]
**Describe the bug**
Managing a project via terraform 1.4.6 works fine.
same code with terraform 1.5.3 produce the following error:

C:\terraform_1.5.3\Modules>..\terraform apply
module.Artifactory.artifactory_user.build-user: Refreshing state... [id=mydemo-build-system]
module.Artifactory.artifactory_user.admin-user: Refreshing state... [id=e175409]

Terraform used the selected providers to generate the following execution plan. Resource actions are indicated with the following symbols:
  + create

Terraform will perform the following actions:

  # module.Artifactory.project.terraform-project will be created
  + resource "project" "terraform-project" {
      + block_deployments_on_limit = true
      + description                = "mydemo project admin: Terraform Project"
      + display_name               = "mydemo Project"
      + email_notification         = true
      + id                         = (known after apply)
      + key                        = "mydemo"
      + max_storage_in_gibibytes   = 10

      + admin_privileges {
          + index_resources  = true
          + manage_members   = true
          + manage_resources = true
        }

      + member {
          + name  = "e175409"
          + roles = [
              + "Project Admin",
            ]
        }
      + member {
          + name  = "mydemo-build-system"
          + roles = [
              + "Contributor",
            ]
        }
    }

Plan: 1 to add, 0 to change, 0 to destroy.

Do you want to perform these actions?
  Terraform will perform the actions described above.
  Only 'yes' will be accepted to approve.

  Enter a value: yes

module.Artifactory.project.terraform-project: Creating...
╷
│ Error:
│ 400 POST https://artifactory-dev.amat.com/access/api/v1/projects
│ {
│   "errors" : [ {
│     "code" : "BAD_REQUEST",
│     "message" : "Project Quota cannot be lower than `1073741824` bytes. Input: -2147483648"
│   } ]
│ }
│
│   with module.Artifactory.project.terraform-project,
│   on ..\Artifactory_Project\main.tf line 51, in resource "project" "terraform-project":
│   51: resource "project" "terraform-project" {

**Requirements for and issue**
- [ ] A fully functioning terraform snippet that can be copy&pasted (no outside files or ENV vars unless that's part of the issue)
- [ ] Your version of artifactory (you can `curl` it at `$host/artifactory/api/system/version`  7.59.9
- [ ] Your version of terraform 1.5.3

**Expected behavior**
should work as terraform 1.4.6.
changes to max_storage_in_gibibytes produce the same error again

**Additional context**

**Comment by alexhung** [comment]
@samuev Using terraform 1.5.6, I was able to create the follow project with no error:

```sh
alexh@alexh-mac terraform-provider-project % terraform apply

Terraform used the selected providers to generate the following execution plan. Resource actions are indicated with the following symbols:
  + create

Terraform will perform the following actions:

  # project.myproject will be created
  + resource "project" "myproject" {
      + block_deployments_on_limit = false
      + description                = "My Project"
      + display_name               = "My Project"
      + email_notification         = true
      + id                         = (known after apply)
      + key                        = "myproj"
      + max_storage_in_gibibytes   = 10

      + admin_privileges {
          + index_resources  = true
          + manage_members   = true
          + manage_resources = true
        }
    }

Plan: 1 to add, 0 to change, 0 to destroy.

Do you want to perform these actions?
  Terraform will perform the actions described above.
  Only 'yes' will be accepted to approve.

  Enter a value: yes

project.myproject: Creating...
project.myproject: Creation complete after 0s [id=myproj]

Apply complete! Resources: 1 added, 0 changed, 0 destroyed.
```

```sh
alexh@alexh-mac terraform-provider-project % terraform -v
Terraform v1.5.2
on darwin_amd64
+ provider registry.terraform.io/jfrog/project v1.1.17

Your version of Terraform is out of date! The latest version
is 1.5.6. You can update by downloading from https://www.terraform.io/downloads.html
```

**Comment by alexhung** [comment]
@samuev I upgraded `terraform` to latest 1.5.6 and still no error when creating new project.

**Comment by alexhung** [comment]
@chb0github You are on the money. The thought of incorrect integer type crossed my mind briefly but I didn't follow up when I couldn't reproduce. Switching type to `uint64` is definitely the safest course of action.

**Comment by chb0github** [comment]
Yeah, you're going to have to replace `strconv.Atoi` to [strconv.PrseUint](https://pkg.go.dev/strconv#ParseUint) too
https://stackoverflow.com/questions/21532113/golang-converting-string-to-int64
